rootfs/entrypoint.sh

```diff
@@ -16,32 +16,24 @@
 
 set -e
 
-mkdir -p /var/log/nginx
 echo 0 > /tmp/nginx.pid
+# fix directory permissions
 writeDirs=( \
-    /etc/nginx/template \
-    /etc/ingress-controller/ssl \
-    /etc/ingress-controller/auth \
-    /var/log \
-    /var/log/nginx \
-    /tmp \
+    /var/log
+    /var/log/nginx
+    /tmp
```

Comment:
why are we leaving these last three foders here while we remove the others?

Reply:
because entrypoint.sh runs at startup and the removed items from a list could be a volume